chore(dev-docs/ci): update variable names in publish workflows and developer manual to explain CI/CD configurations (#2059)
omegat-org/omegat: src_docs/developer/90.ReleaseProcedure.md

A file is updated with signature. You should keep an original file for backup.

## 9. Build a notarized macOS distribution file locally and publish

Only Apple certified developer can perform this step only on a macOS device.
First, make sure the local JRE is up to date, and you have `codesign` command on your platform.
You can build and install the package image using:

```bash
./gradlew installMacArmDist installMacX64Dist
```

When you have already configured `macCodesignIdentity` project property,

```bash
./gradlew installMacArmSignedDist installMacX64SignedDist
```

This command will build and sign the files for both ARM and x64 architectures.

> After you check all required files are properly processed, create the dmg package and submit it to Apple for notarization.

Reviewer: Apple explicitly says that it is a zip file that needs to be sent for notarization.
Once the zip file contents is notarized, a DMG file can be created and signed.

> After you check all required files are properly processed, create the dmg package and submit it to Apple for notarization.

Author: ```suggestion
After you check all required files are properly processed,  submit the contents to Apple for notarization.
```
